pages/common/gleam.md

```diff
@@ -0,0 +1,36 @@
+# gleam
+
+> The compiler, build tool, package manager and code formatter for Gleam, "a friendly language for building type-safe systems that scale!".
+> More information: <https://gleam.run/>.
```

Comment:
I think this link should be to the documentation of the command that this tldr page describes, instead of to the main page of the project.
```suggestion
> More information: <https://gleam.run/writing-gleam/command-line-reference/>.
```